pyproject.toml

```
@@ -6,7 +6,7 @@ authors = ["Angus Williams <anguswilliams91@gmail.com>"]
 
 [tool.poetry.dependencies]
 python = ">=3.7.1,<4.0"
-numpyro = "^0.6.0"
+numpyro = ">=0.6.0"
```

Just suggesting adding a range here to avoid any breaking changes that might happen in the future:

```suggestion
numpyro = ">=0.6.0, <1.0"
```